packages/fbtee/ReactTypes.d.ts

```diff
@@ -11,6 +11,14 @@ enum IntlVariations {
   GENDER_UNKNOWN = 3,
 }
 
+enum GenderConst {
```

Comment:
this wasn't defined before, which resulted in the type being `any`

Reply:
It's ok to import from lib since we need the build step to run `pnpm test` anyway.